pkgs/tools/filesystems/goofys/default.nix

```diff
@@ -30,7 +30,8 @@ buildGoModule {
     description = "A high-performance, POSIX-ish Amazon S3 file system written in Go.";
     license = [ lib.licenses.mit ];
     maintainers = [ lib.maintainers.adisbladis ];
-    broken = stdenv.isDarwin; # needs to update gopsutil to at least v3.21.3 to include https://github.com/shirou/gopsutil/pull/1042
+    # does not build with go 1.17
+    broken = true;
```

Comment:
ping @adisbladis , I don't think you've been notified?

Reply:
Thanks for the ping. Fixed in https://github.com/NixOS/nixpkgs/pull/176255.